src/main/java/generationExamples/ExamplesFromFileGenerationImpl.java

```diff
@@ -0,0 +1,72 @@
+package generationExamples;
+
+import main.Configurations;
+import main.Main;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+public class ExamplesFromFileGenerationImpl implements EquationPattern {
+    private final static String SPLITTER = " ";
```

Comment:
`NUMBERS_SPLITTER_IN_STRING` would be better as a name for variable

Reply:
I see. 
fixed